.github/lstm_for_th_my.json

```diff
@@ -1,15 +0,0 @@
-// © 2021 and later: Unicode, Inc. and others.
```

Comment:
Why are these files being removed? It would be good not to touch them. We can disable the actual workflows instead.